packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -252,16 +252,16 @@ const createEstate = async estate => {
     // See more: https://github.com/tasitlabs/TasitSDK/issues/253
     action.on("confirmation", message => {});
 
-    action.on("error", message => {
-      const { error } = message;
-      console.log(error.message);
+    action.on("error", error => {
+      const { message } = error;
+      console.log(message);
       action.unsubscribe();
       reject();
     });
 
-    estateContract.on("error", message => {
-      const { error } = message;
-      console.log(error.message);
+    estateContract.on("error", error => {
+      const { message } = error;
+      console.log(message);
```

Comment:
Should it be `console.warn`?

Reply:
Yes, sounds good.

Reply:
Solve on https://github.com/tasitlabs/TasitSDK/pull/379

Reply:
Great, thanks! Resolving this.